src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
Hi, changed this to 'angular.module('patternfly.charts').constant('c3ChartDefaults', {...' and renamed file accordingly.'